src/applications/financial-status-report/components/SpouseAdditionalIncomeCheckList.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { setData } from 'platform/forms-system/src/js/actions';
+import { otherIncome } from '../constants/checkboxSelections';
+import Checklist from './utils/CheckList';
+
+const SpouseAdditionalIncomeCheckList = () => {
+  const dispatch = useDispatch();
+  const formData = useSelector(state => state.form.data);
+
+  const { additionalIncome } = formData;
+  const { spAddlIncome = [] } = additionalIncome.spouse;
```

Comment:
In the event that additionalIncome isn't inside of form data for some reason we might want to update this to something like:

```
const { additionalIncome } = formData ?? {};
const { spAddlIncome = [] } = additionalIncome?.spouse ?? {};
  ```

Reply:
Good call, I just added those and am rebuilding the pr now before merge.